lib/charms/layer/apache_zeppelin.py

```diff
@@ -115,6 +117,29 @@ def setup_zeppelin_config(self):
         if not zeppelin_site.exists():
             (self.dist_config.path('zeppelin_conf') / 'zeppelin-site.xml.template').copy(zeppelin_site)
 
+    def setup_init_scripts(self):
+        if host.init_is_systemd():
+            template_path = '/etc/systemd/system/zeppelin.service'
+            template_name = 'systemd.conf'
+        else:
+            template_path = '/etc/init/zeppelin.conf'
+            template_name = 'upstart.conf'
+        if os.path.exists(template_path):
+            os.remove(template_path)
```

Comment:
The last commit should address this comment. We keep the last config with a .backup extension.